modules/core/src/main/java/org/apache/ignite/internal/processors/query/QueryTypeDescriptorImpl.java

```diff
@@ -721,6 +726,18 @@ else if (F.eq(idxField, valueFieldAlias()) || F.eq(idxField, VAL_FIELD_NAME)) {
                 }
                 else if (coCtx.kernalContext().cacheObjects().typeId(propType.getName()) !=
                     ((BinaryObject)propVal).type().typeId()) {
+                    // Check for classes/enums implementing indexed interfaces
```

Comment:
Please, format this code with our code guidelines. There are some issues:
1. Comments should be end with a point
2. There should be a new line before `try` and `if`
3. There should not be a braces around single line with `continue`.

Reply:
https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines